test_scripts/API/AutoCompleteList/009_ATF_SetGlobalProperties_P_AutoCompleteList_ArrayOfLowerBound.lua

```diff
@@ -0,0 +1,79 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = true, resultCode = "SUCCESS"> to mobile app
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param param to HMI
```

Comment:
@Aleksandriys,
1. Description currently looks like "expected result". Please try to use description describing case: "Case when ..."
2. please update description with particular resultCode from HMI (please specify "SUCCESS") and not <resultCode_received_from _HMI>